cmd/admin-service-status.go

```diff
@@ -63,10 +90,8 @@ func (u serviceStatusMessage) String() string {
 		humanize.IBytes(uint64(u.StorageInfo.Total)),
 		humanize.IBytes(uint64(u.StorageInfo.Free)),
 	)
-	if u.StorageInfo.Backend.Type == madmin.XL {
-		msg += fmt.Sprintf(" Online Disks: %d, Offline Disks: %d\n",
-			u.StorageInfo.Backend.OnlineDisks,
-			u.StorageInfo.Backend.OfflineDisks)
+	if v, ok := u.StorageInfo.Backend.(xlBackend); ok {
+		msg += fmt.Sprintf(" Online Disks: %d, Offline Disks: %d\n", v.OnlineDisks, v.OfflineDisks)
```

Comment:
Can we also choose a color combination for this? You can choose something which is closer to `mc ls`